lib/index.ts

```diff
@@ -1,21 +1,23 @@
-import { JellyfishPluginBase } from '@balena/jellyfish-plugin-base';
-import { cards } from './cards';
+import { PluginDefinition } from '@balena/jellyfish-worker';
 import { actions } from './actions';
+import { contracts } from './contracts';
+export * as testUtils from './test-utils';
 
 // tslint:disable-next-line: no-var-requires
 const { version } = require('../package.json');
 
 /**
  * The default Jellyfish plugin.
  */
-export class DefaultPlugin extends JellyfishPluginBase {
-	constructor() {
-		super({
-			slug: 'jellyfish-plugin-default',
-			name: 'Default Jellyfish Plugin',
-			version,
-			cards,
-			actions,
-		});
-	}
-}
+export const defaultPlugin = (
+	definition: Partial<PluginDefinition> = {},
```

Comment:
Why allow users to override entries?

Reply:
I think I pulled this from your tests when you updated the plugin interface in worker. But yeah, now that I take another look, we shouldn't really allow this. Will remove it!

Reply:
Done! :rocket:

Reply:
> I think I pulled this from your tests when you updated the plugin interface in worker

Ya in that case each test may override whatever they want for the test